omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -37,6 +37,21 @@
   node.consume_attributes({"previous_run" => old_config['private_chef']})
 end
 
+# Do a sanity check to make sure both SAML and LDAP not enabled at the same time
+chef_manage = { 'chef_manage' => {} }
+if ::File.exists?('/var/opt/chef-manage/etc/chef-manage-running.json')
+  chef_manage = JSON.parse(IO.read("/var/opt/chef-manage/etc/chef-manage-running.json"))
+  node.consume_attributes({"chef_manage" => chef_manage['chef-manage']})
+end
+
+ldap_enabled = !(node['private_chef']['ldap'].nil? || node['private_chef']['ldap'].empty?)
```

Comment:
Apologies, in my original comment I wasn't clear.  Lines 40-45 make sense here; but lines 48-53 should stay in the their original location since this check will only work after line 110 of this file.